src/main/java/com/example/onlinebookshop/dto/book/BookSearchParametersDto.java

```diff
@@ -0,0 +1,36 @@
+package com.example.onlinebookshop.dto.book;
+
+import static com.example.onlinebookshop.constants.BookDtoConstants.AUTHOR_DESCRIPTION;
```

Comment:
After you refactored code according my comment, I think maybe it's better to use
import static com.example.onlinebookshop.constants.BookDtoConstants.*; ? Or you can't pass checkstyle in this way?

Reply:
Unfortunately, yes, *-style import is not allowed by mate checkstyle. It this case, what do we choose? Constants.SOME_CONSTANT style or just many static imports?